dtfs-central-api/src/cron-scheduler-jobs/create-utilisation-reports/index.test.ts

```diff
@@ -70,29 +107,23 @@ describe('scheduler/jobs/create-utilisation-reports', () => {
 
     it('does not try to create any utilisation reports when reports for all banks in the current period already exist', async () => {
       // Arrange
-      const bank = {
-        id: '123',
-        name: 'Test bank',
-        isVisibleInTfmUtilisationReports: true,
-        utilisationReportPeriodSchedule: MONTHLY_REPORT_PERIOD_SCHEDULE,
-      } as Bank;
-      jest.mocked(getAllBanks).mockResolvedValue([bank]);
-
-      const existingReport = UtilisationReportEntityMockBuilder.forStatus(REPORT_NOT_RECEIVED).withBankId(bank.id).build();
+      jest.mocked(getAllBanks).mockResolvedValue([mockBank]);
+
+      const existingReport = UtilisationReportEntityMockBuilder.forStatus(REPORT_NOT_RECEIVED).withBankId(mockBank.id).build();
       findOneByBankIdAndReportPeriodSpy.mockResolvedValue(existingReport);
 
       // Act
       await createUtilisationReportForBanksJob.task(new Date());
 
       // Assert
-      expect(findOneByBankIdAndReportPeriodSpy).toHaveBeenCalledWith(bank.id, mockReportPeriod);
+      expect(findOneByBankIdAndReportPeriodSpy).toHaveBeenCalledWith(mockBank.id, mockReportPeriod);
```

Comment:
Can we add a `toHaveBeenCalledTimes(1)` assertion please

Reply:
done